src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -38,6 +38,7 @@
 [ReadsComponent(typeof(Engulfable))]
 [ReadsComponent(typeof(MicrobeColony))]
 [ReadsComponent(typeof(WorldPosition))]
+[ReadsComponent(typeof(StrainAffected))]
```

Comment:
This addition makes it necessary to check with the generate threaded run script that this doesn't invalidate the current order.

Reply:
It usually crashes if RunsAfter/Before has some conflicts but this time it also crashes and I don't know why.

Reply:
What happens? If the generate threaded fails to run, then that means this is logically incorrect change. And it only works in the game as it is not enforced but systems may be corrupting the data they are working accidentally simultaneously.

Reply:
/root/godot/modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/ExceptionUtils.cs:112 - System.ArgumentException: An item with the same key has already been added. Key: EditorComponentBase`1[ICellEditorData]

I tried updating libraries before that but it didn't do anything

Reply:
That's just a godot engine bug, that in my understanding causes nothing. You *can* restart the godot editor to get the error to go away for a little bit.